xcodeproject/xcodeproj/target_dependency_test.go

```diff
@@ -1,78 +0,0 @@
-package xcodeproj
```

Comment:
This test is no longer relevant as we no longer parse the target dependencies, only an ID reference. That is tested in `target_test.go`